.github/workflows/oscar-noinclude.yml

```
@@ -0,0 +1,88 @@
+name: OscarCI (no include block)
+
+on:
+   pull_request:
+     branches:
+       - master
+
+jobs:
+  generatematrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    env:
+      PR_NUMBER: ${{github.event.number}}
+    steps:
+    - uses: actions/checkout@v2.1.0
+    - name: "Set up Julia"
+      uses: julia-actions/setup-julia@v1
+      with:
+        version: '~1.6.0-0'
+    - name: OscarDevTools - CI
+      if: github.repository == 'oscar-system/OscarDevTools.jl'
+      run: |
+        julia --project=oscar-dev -e "using Pkg;
+                                      Pkg.add(path=\".\");
+                                      Pkg.instantiate();"
+    - name: fetch OscarDevTools
+      if: github.repository != 'oscar-system/OscarDevTools.jl'
+      run: |
+        julia --project=oscar-dev -e "using Pkg;
+                                      Pkg.add(\"OscarDevTools\");
+                                      Pkg.instantiate();"
+    - id: set-matrix
+      run: |
+        julia --project=oscar-dev -e "using OscarDevTools.OscarCI;
+                 ciprefs = parse_meta(\"OscarCI-noinclude.toml\");
```

```suggestion
                 ciprefs = parse_meta(\"test/meta/OscarCI-noinclude.toml\");
```
this needs the full path to the toml, only the default `OscarCI.toml` has a symlink in the top directory.

---

Thanks, could have taken me a few minutes figuring this out.